Wikipedia/Code/ScrollableEducationPanelViewController.swift

```diff
@@ -131,7 +131,8 @@ class ScrollableEducationPanelViewController: UIViewController, Themeable {
         
         reset()
         primaryButton.titleLabel?.wmf_configureToAutoAdjustFontSize()
-        secondaryButton.titleLabel?.wmf_configureToAutoAdjustFontSize(numberOfLines: 2)
+        secondaryButton.titleLabel?.numberOfLines = 2
+        secondaryButton?.titleLabel?.adjustsFontSizeToFitWidth = true
```

Comment:
Oh is `adjustsFontSizeToFitWidth` now working if `numberOfLines` is greater than 1? iirc it didn't on iOS 11.2 and before...

Reply:
11.2
<img width="459" alt="screen shot 2018-04-10 at 8 03 44 pm" src="https://user-images.githubusercontent.com/9299317/38589659-5335c3ce-3cfa-11e8-8a28-d6dca284209d.png">

10.3
<img width="503" alt="screen shot 2018-04-10 at 8 05 08 pm" src="https://user-images.githubusercontent.com/9299317/38589686-872fd368-3cfa-11e8-8095-7a753c90c9ab.png">

Reply:
One quick thing to check... 

In your screenshots I see the bottom button text wrapped to 2 lines, but did it change the font size to do so (ala `adjustsFontSizeToFitWidth`)? 

To check add a little more text to the `No, remove articles...` string such that it would push it to 3 lines and see if the text gets a little smaller.

Reply:
Thanks for checking. That last sentence was hurtful :(